test/core/auth/test_tokens.py

```diff
@@ -14,21 +16,31 @@
 
 def test_get_user_from_token():
     """Test user generated form token"""
-
+    os.environ["FIA_API_API_KEY"] = "1234567"
     user = get_user_from_token(TOKEN)
     expected_user_number = 1234
     assert user.user_number == expected_user_number
     assert user.role == "user"
 
 
+def test_get_user_from_token_api_key():
+    os.environ["FIA_API_API_KEY"] = "1234567"
+    user = get_user_from_token("1234567")
+    expected_user_number = -1
+    assert user.user_number == expected_user_number
+    assert user.role == "staff"
```

Comment:
Might be intentional but is the role meant to be staff for this test?

Reply:
Yes API calls should be all access